site/docs/en/guides/deployment.md

```diff
@@ -248,24 +242,9 @@ containers:
       export.HADOOP_CONF_DIR: "/etc/hadoop/conf/"                   # Hadoop config dir
       export.HADOOP_USER_NAME: "hadoop"                             # Hadoop user submit on yarn
       export.FLINK_CONF_DIR: "/etc/hadoop/conf/"                    # Flink config dir
```

Comment:
should we add  memory configuration here ?

Reply:
The "memory" information is configured within the optimizer group.